tests/test_base.py

```diff
@@ -0,0 +1,133 @@
+import unittest
```

Comment:
@grzes314 I am only thinking whether to use unittest or pytest. pytest is generally easier for non dev people  Do you feel strongly about unittest ?

Reply:
I think I chose unittest because of its similarity to JUnit which I used before. But I don't insist. I can rewrite these tests in pytest.